### DISCUSSION TITLE
0.4.0 Release

### DISCUSSION THREAD

**Discussion by alienatedsec** [alienatedsec/yi-hack-v5#281] [Announcements]
# You need version `0.3.8` as a minimum to load this release - you can find out more [here ](https://github.com/alienatedsec/yi-hack-v5/releases/tag/0.3.8) and #267 and #279

## What's Changed
* compile.proxychains-ng: properly run configure by @rofl0r in https://github.com/alienatedsec/yi-hack-v5/pull/274
* proxychains-ng bumped to `master` post `v4.16`
* Fixed Proxychain paths and WebUI config page #278 
* Minor amendments to WebUI
* Reworked recording without cloud - fixed #277 
* Fixed Wi-Fi Strength - #42 - thanks to @young747
* Introduced a `./cloud &` process monitoring

## New Contributors
* @rofl0r made their first contribution in https://github.com/alienatedsec/yi-hack-v5/pull/274

**Full Changelog**: https://github.com/alienatedsec/yi-hack-v5/compare/0.3.9...0.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/alienatedsec/yi-hack-v5/releases/tag/0.4.0'>0.4.0 Release</a>.</em>

**Comment by ccw23aseb** [comment]
![image](https://github.com/alienatedsec/yi-hack-v5/assets/158594632/5c0397f9-3ce3-40d4-bdb6-873e01eb8018)

not sure is it just me or...
i always get an error while testing on proxychains as per the screenshot above. tested the CN proxy, no luck so testing active SG proxy just to see if it's working but again, no luck. 
all other features seems to work except proxychains.
FYI, i am using 48CN and i am currently disabled cloud and enabled proxychains-ng.
Appreciate if someone can shed some light on this, thanks.
